cpp/cmake/thirdparty/get_cuvs.cmake

```diff
@@ -74,7 +74,7 @@ endfunction()
 # CPM_cuvs_SOURCE=/path/to/local/cuvs
 find_and_configure_cuvs(VERSION          ${CUML_MIN_VERSION_cuvs}
       FORK             rapidsai
-      PINNED_TAG       branch-${CUML_BRANCH_VERSION_cuvs}
+      PINNED_TAG       fea-2408-rbc
```

Comment:
Can we either block this PR or create an issue to track unpinning, please?

Reply:
Your blocking review is fine, I will let you know when I unpin.

Reply:
Unpinned https://github.com/rapidsai/cuml/pull/6644/commits/9192e3d727a21c4f6e0843bb8da4a5319fdc995f